### ISSUE TITLE
Facebook messages not syncing

### ISSUE THREAD

**Issue by mind12** [Bjornskjald/miscord#202]
_If you have a question, ask on the [Miscord Support server](https://discord.gg/DkmTvVz) instead_  
## Expected Behavior

Message sync

## Actual Behavior

2|miscord  | info login Launching Miscord v3.1.1
2|miscord  | info login Logging in...
2|miscord  | info login Log level: info
2|miscord  | info login Logging in to Discord...
2|miscord  | info login Logged in to Discord
2|miscord  | info login Got Discord channels list
2|miscord  | info login Logging in to Facebook...

2|miscord  | info
2|miscord  |  login Logged in
2|miscord  | info
2|miscord  |  discordListener Got a Discord message
2|miscord  | info
2|miscord  |  
2|miscord  | messengerListener
2|miscord  |  Got a Messenger message
2|miscord  | You have triggered an unhandledRejection, you may have forgotten to catch a Promise rejection:
2|miscord  | DiscordAPIError: Invalid Form Body
2|miscord  | username: Must be between 2 and 32 in length.
2|miscord  |     at item.request.gen.end (/opt/miscord/node_modules/discord.js/src/client/rest/RequestHandlers/Sequential.js:71:65)
2|miscord  |     at then (/opt/miscord/node_modules/snekfetch/src/index.js:213:21)
2|miscord  |     at <anonymous>
2|miscord  |     at process._tickCallback (internal/process/next_tick.js:182:7)
2|miscord  | info
2|miscord  |  messengerListener Got a Messenger message
2|miscord  | (node:32272) UnhandledPromiseRejectionWarning: DiscordAPIError: Invalid Form Body
2|miscord  | username: Must be between 2 and 32 in length.
2|miscord  |     at item.request.gen.end (/opt/miscord/node_modules/discord.js/src/client/rest/RequestHandlers/Sequential.js:71:65)
2|miscord  |     at then (/opt/miscord/node_modules/snekfetch/src/index.js:213:21)
2|miscord  |     at <anonymous>
2|miscord  |     at process._tickCallback (internal/process/next_tick.js:182:7)
2|miscord  | (node:32272) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 4)
2|miscord  | (node:32272) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

## Steps to Reproduce the Problem

  1. Write a message to facebook chat
  1.
  1.

## Specifications

  - Platform: Linux
  - Version (`miscord -v`): 3.1.1
  - Node.js version (`node -v`): v9.11.1

**Comment by mind12** [comment]
Went back to v3.0.14, no problem with that version.

**Comment by Bjornskjald** [comment]
Did you update to 3.1.1? That issue was only on 3.1.0

**Comment by mind12** [comment]
I attached the logs, yes I used v3.1.1:

**2|miscord | info login Launching Miscord v3.1.1**
2|miscord | info login Logging in...
2|miscord | info login Log level: info
2|miscord | info login Logging in to Discord...
2|miscord | info login Logged in to Discord
2|miscord | info login Got Discord channels list
2|miscord | info login Logging in to Facebook...

**Comment by Bjornskjald** [comment]
Can you change log level to silly and post more detailed logs?  
(if you can reproduce the issue of course)

**Comment by mind12** [comment]
Ok I got the source of the error, long and special messenger nicknames causes that the specific user's messages don't sync. The bot most be restarted after changing back the nickname to a shorter version. After that it works.

I had to revert back to v3.0.14. I don't know which characters cause the problem and I can't force all the messenger users to change nickname because of this.

Attached the silly logs and a nickname example.

![image](https://user-images.githubusercontent.com/22614709/38833620-5288356c-41c6-11e8-8b0c-9ae9af98e9eb.png)


[silly_miscord.txt](https://github.com/Bjornskjald/miscord/files/1917160/silly_miscord.txt)
